### ISSUE TITLE
Machine Learning model for Diabetes Prediction

### ISSUE THREAD

**Issue by simranquirky** [prathimacode-hub/PyAlgo-Tree#145]
the model takes input like blood pressure, gender, age, Plasma concentration etc, and predicts whether the person needs treatment or not

**Comment by prathimacode-hub** [comment]
Hmm, but follow the issue template provided to get issue assigned. By the way, are you a LGMSoC participant, DCP participant or contributor? @simranquirky

**Comment by simranquirky** [comment]
> Hmm, but follow the issue template provided to get issue assigned. By the way, are you a LGMSoC participant, DCP participant or contributor? @simranquirky 

DCP contributor

**Comment by simranquirky** [comment]
> Hmm, but follow the issue template provided to get issue assigned. By the way, are you a LGMSoC participant, DCP participant or contributor? @simranquirky 

I cannot find the issue template 😅 are you referring to the pull request template?

**Comment by prathimacode-hub** [comment]
No, go through the README of the project you would find it. @simranquirky

**Comment by prathimacode-hub** [comment]
Issue assigned. @simranquirky

**Comment by prathimacode-hub** [comment]
Its been 3 days. 9th is the last date to accept PR's. If you wouldn't do it by then, obviously you can submit later but points won't be counted towards DCP program. @simranquirky

**Comment by simranquirky** [comment]
> Its been 3 days. 9th is the last date to accept PR's. If you wouldn't do it by then, obviously you can submit later but points won't be counted towards DCP program. @simranquirky  

I am working on the model. Hope to complete it before 9th.

**Comment by prathimacode-hub** [comment]
Update? @simranquirky

**Comment by prathimacode-hub** [comment]
Time is up for the program. Unfortunately i have to remove the label. But you submit it ASAP hopefully with your PR. @simranquirky

**Comment by prathimacode-hub** [comment]
Any update? @simranquirky

**Comment by prathimacode-hub** [comment]
Updates please? @simranquirky

**Comment by prathimacode-hub** [comment]
It's been 14 days since last message. Kindly update the progress and issue can't be held for this long. @simranquirky

**Comment by prathimacode-hub** [comment]
Update please? @simranquirky

**Comment by deepthi1107** [comment]
please do assign me mam @prathimacode-hub

**Comment by prathimacode-hub** [comment]
Issue assigned to @deepthi1107

**Comment by prathimacode-hub** [comment]
Update please? @parvatijay2901
